components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue

```diff
@@ -99,19 +100,36 @@ const dprops = defineProps<{
 const { decimals, unit } = useChain()
 const { urlPrefix, tokenId, assets } = usePrefix()
 
-const { data, loading } = useGraphql({
+const interaction =
+  urlPrefix.value === 'rmrk2'
+    ? dprops.interactions.filter((i) => i !== 'MINTNFT' && i !== 'CONSUME')
+    : dprops.interactions
+
+const { data, loading, refetch } = useGraphql({
   queryName: 'itemEvents',
   clientName: urlPrefix.value,
   variables: {
     id: dprops.nftId,
-    interaction:
-      urlPrefix.value === 'rmrk2'
-        ? dprops.interactions.filter((i) => i !== 'MINTNFT' && i !== 'CONSUME')
-        : dprops.interactions,
+    interaction,
     limit: 100,
   },
 })
 
+useSubscriptionGraphql({
+  query: `
+  events (
+    where: { nft: { id_eq: "${dprops.nftId}" }, interaction_in: [${interaction}] }
+    orderBy: timestamp_DESC
+    limit: 5
+  ) {
+    id
+    interaction
+    timestamp
+    meta
+  }`,
```

Comment:
can this use a `.graphql` file pls?

it is very easy to introduce bug if it passed like a string